### ISSUE TITLE
incorrect alt text error

### ISSUE THREAD

**Issue by jmankoff** [a11yhood/a11yhood.github.io#186]
When uploading a new product, if it doesn't have an image, it errors out saying the product can't be submitted because alt text is missing. It should either say an image is missing, or allow upload with no alt text when there is no image.
